external/deployments/rskMainnet/CommunityIssuance_Implementation.json

```diff
@@ -0,0 +1,607 @@
+{
+    "address": "0x5059A057df377436Fc9b68369b696A23cc61Ecc3",
```

Comment:
the deployment files CommunityIssuance_Implementation and mmunityIssuance_Procy are not used, pls remove